qutebrowser/browser/commands.py

```diff
@@ -985,6 +987,32 @@ def quickmark_load(self, name, tab=False, bg=False, window=False):
         url = objreg.get('quickmark-manager').get(name)
         self._open(url, tab, bg, window)
 
+    @cmdutils.register(instance='command-dispatcher', name='select-follow',
+                       scope='window')
+    def select_follow(self, tab=False):
+        """Follow the selected text.
+
+        Args:
+            tab: Load the selected link in a new tab.
+        """
+        widget = self._current_widget()
+        if QWebSettings.globalSettings().testAttribute(
+                QWebSettings.JavascriptEnabled):
+            if tab:
+                widget.page().open_target = usertypes.ClickTarget.tab
+            widget.page().currentFrame().evaluateJavaScript(
+                'window.getSelection().anchorNode.parentNode.click()')
+        else:
+            try:
+                selected_element = ElementTree.fromstring(
+                    '<html>' + widget.selectedHtml() + '</html>').find('a')
```

Comment:
It is because I was getting an error when the `selectedHtml` contained more than one element. For instance, when I select more than a link, I will get something like `<a href="...">...</a><span>...</span>` which throws an error in `ElementTree.fromstring`.